benchmarks/libpcap_fuzz_both/build.sh

```diff
@@ -23,12 +23,15 @@ cd build
 cmake ..
 make
 
+# Add required libs for AFL++.
+[[ $FUZZER == aflplusplus ]] && export EXTRA_LIBS='/usr/lib/x86_64-linux-gnu/libdbus-1.a /lib/x86_64-linux-gnu/libsystemd.so.0'
```

Comment:
This is very bizarre. I'd prefer it if this change could somehow be done in the aflplusplus fuzzer.py instead of here.

Reply:
Yeah, I was thinking about the same trade-off.
I put it here because I assumed that it is more likely that 1) other fuzzers will need this on this benchmark, and less likely that 2) `aflp++` will need this on a different benchmark.

For 1), we can simply add another fuzzers in the first operand of `&&`.
For 2), yes, it is better to put the line in `afl++`.

Reply:
SG! Moved to `afl++`'s `fuzzer.py`.